pallets/ethereum-checked/src/tests.rs

```diff
@@ -21,14 +21,14 @@
 use super::*;
 use mock::*;
 
-use astar_primitives::ethereum_checked::MAX_ETHEREUM_TX_INPUT_SIZE;
+use astar_primitives::ethereum_checked::EthereumTxInput;
 use ethereum::{ReceiptV3, TransactionV2 as Transaction};
-use frame_support::{assert_noop, assert_ok, traits::ConstU32};
+use frame_support::{assert_noop, assert_ok};
 use sp_runtime::DispatchError;
 
-fn bounded_input(data: &'static str) -> BoundedVec<u8, ConstU32<MAX_ETHEREUM_TX_INPUT_SIZE>> {
-    BoundedVec::<u8, ConstU32<MAX_ETHEREUM_TX_INPUT_SIZE>>::try_from(hex::decode(data).unwrap())
-        .unwrap()
+fn bounded_input(data: &'static str) -> EthereumTxInput {
+    EthereumTxInput::try_from(hex::decode(data).expect("invalid input hex"))
+        .expect("input too large")
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `bounded_input` function has been modified to use the `EthereumTxInput` type instead of `BoundedVec<u8, ConstU32<MAX_ETHEREUM_TX_INPUT_SIZE>>`. This change aligns with the updated codebase. However, there is a missing import for `ethereum_checked::EthereumTxInput` on line 24. Please add the import statement:

```diff
+use astar_primitives::ethereum_checked::EthereumTxInput;
```

Additionally, the `try_from` and `expect` calls on lines 30-31 can be simplified using the `unwrap` method. Here's the updated code:

```diff
fn bounded_input(data: &'static str) -> EthereumTxInput {
-    EthereumTxInput::try_from(hex::decode(data).expect("invalid input hex"))
-        .expect("input too large")
+    EthereumTxInput::from(hex::decode(data).unwrap())
 }
```

Please make these changes in the code.


<!-- This is an auto-generated comment by OSS CodeRabbit -->